addons/im_livechat/static/tests/go_to_oldest_unread_thread.test.js

```diff
@@ -92,6 +96,104 @@ test("tab on discuss composer goes to oldest unread livechat", async () => {
     await contains(".o-mail-DiscussSidebarChannel.o-active", { text: "Visitor 12" });
 });
 
+test("Tab livechat picks ended livechats last", async () => {
+    const pyEnv = await startServer();
+    const guestIds = pyEnv["mail.guest"].create([
+        { name: "Visitor 1" },
+        { name: "Visitor 2" },
+        { name: "Visitor 3" },
+        { name: "Visitor 4" },
+        { name: "Visitor 5" },
+    ]);
+    const livechatChannelId = pyEnv["im_livechat.channel"].create({
```

Comment:
Nitpick: would name Visitor from 0 to 4, so it matches the array index. This helps test readability and understanding.